### DISCUSSION TITLE
Download OpenAPI spec button

### DISCUSSION THREAD

**Discussion by thelinuxlich** [scalar/scalar#313] [Ideas]
Would be really useful, as every other project does it too

**Comment by marclave** [comment]
This is a great idea, let's figure this out tomorrow @cameronrohani

**Comment by marclave** [comment]
@thelinuxlich the wonderful @hanspagel shipped this today! Can pull latest main and test it out ✨ 

https://github.com/scalar/scalar/pull/362
